client/src/components/InputElements/VisualFiltersDropdown.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  Button,
+  Intent,
+  Menu,
+  MenuItem,
+  Popover,
+  Checkbox,
+} from '@blueprintjs/core';
+import React, { memo } from 'react';
+
+import { usePopover } from '~/ui/hooks/usePopover';
+
+interface Props {
+  showHost: boolean;
+  onShowHostToggle: () => void;
+  showKubeDns: boolean;
+  onShowKubeDnsToggle: () => void;
```

Comment:
I would keep all events callbacks optional not be too obsessive in `child - parent` relation. It's parent deal to catch that event or not.